README.md

```diff
@@ -76,6 +76,8 @@ that represents availability by zone.
 See Github [Issues](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues)
 
 ## Driver Deployment
+**Note:** With a Kubernetes Engine (GKE) cluster, we recommend automatically deploying the driver through cluster configuration instead of deploying manually using scripts in this repository. The automatic deployment mechanism is currently alpha, and the user guide can be found [here](https://docs.google.com/document/d/1EUqP73wD5XcxezHjT7vjZ15R7rRc_qPEHSu9JiEL8ZY/edit?usp=sharing).
```

Comment:
We should not be making these documents publicly readable.  Let's wait until it's available outside of the whitelist before adding a note here